bbb-libreoffice/assets/convert-remote.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/bash
+# This is a sample script - adjust it per your need
+# 1 - setup a server with JOD-CONVERTER-REST ( docker run --memory 512m --rm -p 8080:8080 eugenmayer/jodconverter:rest )
```

Comment:
I know I'm (very) late to the party, but this doesn't seem like a good suggestion, because this project is probably not a good idea to use in a commercial context: https://github.com/EugenMayer/officeconverter/issues/5#issuecomment-670350480